config/settings.yml

```diff
@@ -558,6 +559,13 @@ mhv:
     host: fake-host
     app_token: fake-app-token
     x_auth_key: fake-x-auth-key
+  account_creation:
+    mock: true
+    host: https://mhv-intb-api.myhealth.va.gov
+    sts:
+      service_account_id: c34b86f2130ff3cd4b1d309bc09d8740
+      issuer: http://localhost:3000
+      private_key_path: spec/fixtures/sign_in/sts_client.pem
```

Comment:
Do you think we need this? Or do you see any reason not to use `Settings.sign_in.sts_client`?

Reply:
yeah we can. i went back and forth. i didn't know how decoupled we wanted to make it

Reply:
oh, so i already did switch to use `Settings.sign_in.sts_client.private_key_path`. I just forgot to remove it from here

Reply:
will the issuer be the same across all clients on vets-api?